frontend/public/kinds.ts

```diff
@@ -21,23 +20,11 @@ export const connectToModel = connect(
   },
 );
 
-type WithPluralProps = {
-  kindObj?: K8sKind;
-  modelRef?: K8sResourceKindReference;
-  kindsInFlight?: boolean;
-};
-
-export type ConnectToPlural = <P extends WithPluralProps>(
-  C: React.ComponentType<P>,
-) => React.ComponentType<Omit<P, keyof WithPluralProps>> & {
-  WrappedComponent: React.ComponentType<P>;
-};
-
```

Comment:
Removed these types because they are incorrect, overly complex, and deprecated. Fixing the ts errors they cause are not worthwhile.